### PR TITLE
Update data.js

### PR DESCRIPTION
**Pull request by ashishalf**
**Adding a new project ChatGPT Clone using Html, Css and Js with OpenAI API**

I would like to propose the implementation of a ChatGPT clone using HTML, CSS, and JS, utilizing the OpenAI API for natural language processing capabilities. This project aims to create an interactive chatbot interface that simulates conversations with users in a similar manner to OpenAI's ChatGPT model.

The clone would be designed to handle user queries and respond with relevant and contextually appropriate answers. By leveraging the OpenAI API, the chatbot would be capable of generating human-like responses, making the conversations more engaging and natural.

The implementation would involve developing the front-end components using HTML, CSS, and JS to create an intuitive and user-friendly interface. The OpenAI API would be integrated to handle the backend processing of user input and generating appropriate responses.

**Key features of the ChatGPT clone would include:**

1. Input textbox: Users can enter their queries or messages into the chat interface.
2. Response display: The chatbot will provide real-time responses that are generated based on the user input and processed by the OpenAI API.
3. Contextual understanding: The chatbot will maintain context throughout the conversation to provide accurate and meaningful responses.
4. User-friendly design: The interface will be visually appealing, responsive, and easy to navigate, ensuring a smooth user experience.

By creating this ChatGPT clone, we aim to provide a practical example of how the OpenAI API can be integrated into web applications to enhance user interactions and provide intelligent responses. This project would serve as a valuable learning resource and showcase the capabilities of the OpenAI technology.

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -1,4 +1,14 @@
 exports.projectsData = [
+    {
+    id: 206,
+    type: "video",
+    title: "Create Your Own ChatGPT in HTML CSS and JavaScript",
```
Comment (Xtremilicious) [inline data.js:5]:
```suggestion
    title: "Create Your Own ChatGPT Clone in HTML CSS and JavaScript",
```

File: data.js [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+    category: ["web-dev"],
+    tech: ["HTML","CSS","JavaScript", "OpenAI API", "API"],
+    datePublished: "July 12, 2023",
+    projectURL:"https://youtu.be/atKtG29iroY",
+
```
Comment (Xtremilicious) [inline data.js:10]: Please include a difficulty level for this entry. This feature will be valuable for users when they start filtering by difficulty, a functionality that will soon be available on ProjectLearn.

Add a new key `difficulty` with one of these values: `easy`, `intermediate` or `advanced`

**Review by Xtremilicious — CHANGES_REQUESTED** [review]
Thanks for your contribution!

**Comment by ashishalf** [comment]
Added difficulty.

**Review by Xtremilicious — APPROVED** [review]

**Pull request merged**